Sources/ATTNSDK.m

```diff
@@ -206,6 +206,15 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
         [UIApplication.sharedApplication openURL:url];
         decisionHandler(WKNavigationActionPolicyCancel);
         return;
+    } else if ([navigationAction.request.URL.scheme isEqualToString:@"http"] || [navigationAction.request.URL.scheme isEqualToString:@"https"]) {
```

Comment:
should we make this a case insensitive check like we do for android?

Reply:
Done!